client/src/App.vue

```diff
@@ -22,6 +22,8 @@ onMounted(async (): Promise<void> => {
   SplashScreen.hide();
 
   const config = await storageManager.retrieveConfig();
+  // temporary till the dark mode is working
+  config.theme = 'light';
```

Comment:
Would be better to change it in `storageManager.ts` directly, like in the `DEFAULT_CONFIG` variable.